documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1432,6 +1432,10 @@ Note: Depending on the `_hashAlgorithm_` used, the `_salt_` selected and the act
 |`adaptive`
 | Time, date, and timestamps can be represented with different kinds of precision, including: `adaptive` (the default) captures the time and timestamp values exactly as in the database using either millisecond, microsecond, or nanosecond precision values based on the database column's type; or `connect` always represents time and timestamp values using Kafka Connect's built-in representations for Time, Date, and Timestamp, which uses millisecond precision regardless of the database columns' precision. See {link-prefix}:{link-sqlserver-connector}#sqlserver-temporal-values[temporal values].
 
+|[[sqlserver-property-include-schema-changes]]<<sqlserver-property-include-schema-changes, `include.schema.changes`>>
```

Comment:
@gunnarmorling  Should the doc for the Db2 connector also have this property?

Reply:
Yes it should.

Reply:
I just added that with the latest commit.